docs/en/stack/ml/nlp/ml-nlp-elser.asciidoc

```diff
@@ -433,9 +462,48 @@ on two data sets and different hardware configurations. Your data set has an
 impact on the model performance. Run tests on your own data to have a more 
 realistic view on the model performance for your use case.
 
+[discrete]
+==== Summary
+
+Overall the platform specific V2 model ingested at a max rate of 25.90 docs/s, 
+compared with the ELSER V1 max rate of 13.65 docs/s from the ELSER V1 benchamrk, 
```

Comment:
(optional) While these numbers are accurate, we probably don't want to imply a higher level of consistency than there is. I propose that we round these rates to include no decimal points: 26 doc/s and 14 doc/s respectively to represent the relatively high variability of these values. If you decide to do this rounding, there are a few other numbers below that should also be rounded.

I realize that I included this level of specificity in my report, but upon reflection I think it might not have been the best choice.